### PR TITLE
[Issue- 408] Data Load is failing when no measure is used in create cube command

### PR DESCRIPTION
**Pull request by mohammadshahidkhan**
Loading data with measure column was failing, to fix the same added added the dummy measure when no measure is defined the create table DDL.

Select * from TableName;
Output
image

Must be merged after PR  #425

### CODE REVIEW

**Comment by gvramana** [comment]
retest this please

File: format/src/main/thrift/schema.thrift [hunk 1]
```diff
@@ -85,6 +85,10 @@ struct ColumnSchema{
 	11: optional string aggregate_function;
 
 	12: optional binary default_value;
+	/**
+	* To specify the visibily of the column by default its true
+  **/
+	13: required bool visible = true;
```
Comment (gvramana) [inline format/src/main/thrift/schema.thrift]: Move required field to invisible and make in optional

File: format/src/main/thrift/schema.thrift [hunk 2]
```diff
@@ -85,6 +85,10 @@ struct ColumnSchema{
 	11: optional string aggregate_function;
 
 	12: optional binary default_value;
+	/**
+	* To specify the visibily of the column by default its true
+  **/
```
Comment (jackylk) [inline format/src/main/thrift/schema.thrift]: incorrect indentation

File: format/src/main/thrift/schema.thrift (hunk 1 above)
Comment (jackylk) [inline format/src/main/thrift/schema.thrift]: Agree to @gvramana and I think we should build mechanism for backward compatibility for any change in carbon-format layer, especially adding new **required** field

**Comment by mohammadshahidkhan** [comment]
Fixed Review comments
Build successful with running 277 Testcase
image

**Comment by gvramana** [comment]
Add a check in Carbon Relation to skip invisible columns.

**Comment by mohammadshahidkhan** [comment]
Fixed review comment 
please review the same

**Comment by ravipesala** [comment]
retest this please

**Comment by gvramana** [comment]
retest this please

**Pull request merged**